packages/auth/amplify_auth_cognito/example/integration_test/sign_in_sign_out_test.dart

```diff
@@ -67,6 +67,18 @@ void main() {
       fail('Expected NotAuthorizedException');
     });
 
+    testWidgets(
+        'should throw an NotAuthorizedException if a password is not provided',
```

Comment:
So this will only throw if custom auth is not configured, right? Can we add that to the description? Would be nice to see a success case as well for this.

Reply:
We will need to update our integ tests to use mutliple environments, and add a provision script for them. Gonna work on it as a follow on.